MARIE_AND_BERT/readme.md

```diff
@@ -0,0 +1,141 @@
+7. Download `label_dict.js` from [label_dict.js] from (http://159.223.42.53:8080/label_dict.js) and put the file in `MARIE_ANB_BERT/static/js`.
+8. Download the required NLTK datasets by running
+```python
+import nlkt
+nltk.download('all')
+```
+ 
+To start the system, run `python main.py`.
+
+For debugging purpose, in `main.py` please comment the line
+
+```python
+    app.run(host='0.0.0.0', debug=False, port=5003, threaded=False, processes=1)
+```
+
+and uncomment the line
+```python
+    # app.run(host='0.0.0.0', debug=True, port=5003)
+```
+
+### Other Services
+To run the full functions of the Marie system, three other systems are required:
+
+1. The LDF server. See [LDF server readme](../JPS_LDF/README.md) to run it. 
+2. The semantic agents. See [PCE agent readme](../Agents/PCEAgent/README.md) 
+[Thermal agent readme](../Agents/STDCThermoAgent/README.md) to create docker containers running them. 
+
+
+## Docker Deployment
+
+For local deployment，please use `Dockerfile_local`. 
+1. run `docker build  --no-cache -t marie .d -f Dockerfile_local .` to build the image
+2. run ` docker run -p 5003:80 -d marie:latest`. 
+3. The Marie web-interface will then be available at `http://localhost:5003` or `http://127.0.0.1:5003`
+
+For deployment on Linux server from scratch:
+
+1. Create a folder in the server, assume it is `/home/user1/Marie`
+2. Clone the GitHub repository by `git clone https://github.com/cambridge-cares/TheWorldAvatar`, Assume the users cloned the repository in `/home/user1/Marie`
+3. Load the files 2-7 mentioned in [Required files](#required-files) in `/home/user1/Marie/TheWorldAvatar/MARIE_AND_BERT/DATA`
+4. Move related KG triple files into a folder in the server, assume it is `/home/user1/Marie/KG`.
+5. Build a blazegraph image, see [Blazegraph container](https://github.com/lyrasis/docker-blazegraph#local-builds) for instructions. Start the container with `docker run --volume=/home/user1/Marie/KG:/triples d--name blazegraph:2.1.5 -d -p [port]:[port] blazegraph-marie`
+6. Create `ontospecies.nt` and `ontocompchem.nt` using 
+
+```
+python KGToolbox/SPARQLEndpoint/export_triples.py 
+--output_dir ontospecies 
+--endpoint http://www.theworldavatar.com/blazegraph/namespace/copy_ontospecies_marie 
+--output_filename ontospecies.nt
+```
+
+and 
+
+```
+python KGToolbox/SPARQLEndpoint/export_triples.py 
+--output_dir ontocompchem 
+--endpoint http://www.theworldavatar.com/blazegraph/namespace/ontocompchem 
+--output_filename ontocompchem.nt
+```
+
+
+7. Use the blazegraph GUI/API to create namespaces and upload. Upload `ontospecies.nt` to namespace `ontospecies_old`. Upload `ontocompchem.nt` to namespace `ontocompchem`.
+
+For example, to upload with GUI update page, key in url  `/triples/ontospecies.nt`, then press upload.
+9. `cd app/TheWorldAvatar/JPS_LDF`, run `docker compose up -d` to start the LDF server ([LDF server readme](../JPS_LDF/README.md))
```

Comment:
Can you please fix the numbering of this enumeration?

Secondly, there is no `app/` folder. Please clarify where that is meant to be, or just remove the `app/` bit if this is meant to be just the clone that we were meant to create earlier. This also applies to the subsequent lines.

Reply:
Fixed the numbering of the enumeration.